provision.sh

```diff
@@ -38,32 +43,33 @@ MYSQL=`which mysql`
 PHP=`which php`
 
 # Setup dummy client files
-cd /var/www/etoa && mkdir htdocs/web/build && echo "{}" > htdocs/web/build/manifest.json && echo '{"entrypoints": {"admin": {}}}' > htdocs/web/build/entrypoints.json
+cd /var/www/etoa && mkdir -p htdocs/web/build && echo "{}" > htdocs/web/build/manifest.json && echo '{"entrypoints": {"admin": {}}}' > htdocs/web/build/entrypoints.json
 
-# Install PHP composer dependencies
-cd /var/www/etoa && export COMPOSER_ALLOW_SUPERUSER=1;php composer.phar install --no-interaction
-
-# Install node dependencies and trigger build
-cd /vagrant && yarn install --frozen-lockfile
+# trigger yarn build
 cd /var/www/etoa && yarn run build
 
 # Setup database
 Q0="SET GLOBAL sql_mode=(SELECT REPLACE(@@sql_mode,'ONLY_FULL_GROUP_BY',''));"
 Q1="CREATE DATABASE IF NOT EXISTS etoa;"
-Q2="GRANT USAGE ON *.* TO etoa@localhost IDENTIFIED BY 'etoa';"
+Q2="CREATE USER 'etoa'@'localhost' IDENTIFIED BY 'etoa';"
+#Q2="GRANT USAGE ON *.* TO etoa@localhost IDENTIFIED BY 'etoa';"
```

Comment:
Was auskommentiert ist hier, sonst gerne einfach entfernen. Nachschauen, kann man das sonst immer in der git history.